client/views/add/add.js

```diff
@@ -92,13 +92,18 @@ Template.add.events({
     }
     Meteor.call('addMods', mods, template.data._id, (error, result) => {
       // If the result is an object, there was an error
-      if (typeof result === 'object') {
+      if (typeof result === 'object' && result.length > 0) {
         // Alert the error
         for (let i = 0; i < result.length; i++) {
           if (result[i].name === 'moderators') {
             showModsError('You can only assign 4 moderators per instance.');
             return false;
           }
+          // Check is the server returned error corresponding to the addition of owner as moderator
+          if(result[i].name === 'owner') {
+            showModsError(`${result[i].value} is already an owner of the instance and has the privileges of a moderator.`); // Display the error message
```

Comment:
@singhsanket143 can we move the comment up?

Reply:
@peter-hank I have Updated the comments